debian/postinst

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -eu
+
+g_s_helper_user=g-s-helper
+g_s_helper_home=/var/lib/${g_s_helper_user}
+
+# creating g-s-helper user if it isn't already there
+if ! getent passwd ${g_s_helper_user} >/dev/null; then
+    adduser --system --force-badname --quiet \
+            --no-create-home \
+            --home ${g_s_helper_home} \
+            --shell /usr/sbin/nologin \
+            --group ${g_s_helper_user}
+fi
+
+# create /var/lib/g-s-helper or update ownership of existing directory
+if [ -d ${g_s_helper_home} ]; then
+    chown -R ${g_s_helper_user}:${g_s_helper_user} ${g_s_helper_home}
+else
+    install -d -m0755 -o ${g_s_helper_user} -g ${g_s_helper_user} ${g_s_helper_home}
+fi
+
+# create external apps folders or update ownership of existing directory
+ext_apps_folder=/opt/flatpak-external-apps
+ext_apps_repo=${ext_apps_folder}/repo
+
+if [ -d "${ext_apps_repo}" ]; then
+    chown -R ${g_s_helper_user}:${g_s_helper_user} "${ext_apps_folder}"
+else
+    install -d -m0755 -o ${g_s_helper_user} -g ${g_s_helper_user} "${ext_apps_folder}"
+    install -d -m0755 -o ${g_s_helper_user} -g ${g_s_helper_user} "${ext_apps_repo}"
+fi
+
+# add or modify external apps repo if needed
+ext_apps_repo_name=eos-external-apps
+
+flatpak remote-add --no-gpg-verify ${ext_apps_repo_name} ${ext_apps_repo}
```

Comment:
I think it would be enough for this configuration to only happen in the image builder, since we don't add the other repository from the g-s package itself. No strong feelings either way though.